icons/squares-exclude.svg

```diff
@@ -0,0 +1,18 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M10 8a2 2 0 0 0-2 2" />
+  <path d="M15.73 4A2 2 0 0 0 14 3H5a2 2 0 0 0-2 2v9a2 2 0 0 0 1 1.73" />
+  <path d="M16 10V8h-2" />
+  <path d="M16 14a2 2 0 0 1-2 2" />
+  <path d="M8 14v2h2" />
+  <path d="M8.27 20A2 2 0 0 0 10 21h9a2 2 0 0 0 2-2v-9a2 2 0 0 0-1-1.73" />
```

Comment:
- Replaced curves with arcs.
- Arcified corners.
```suggestion
  <path d="M15 21h-1" />
  <path d="M15.733 4A2 2 0 0 0 14 3" />
  <path d="M21 14v1" />
  <path d="M21 19a2 2 0 0 1-2 2" />
  <path d="M21 9.995a2 2 0 0 0-1-1.727" />
  <path d="M3 10V9" />
  <path d="M3 5a2 2 0 0 1 2-2" />
  <path d="M4 15.737A2 2 0 0 1 3 14" />
  <path d="M8 10a2 2 0 0 1 2-2h5a1 1 0 0 1 1 1v5a2 2 0 0 1-2 2H9a1 1 0 0 1-1-1z" />
  <path d="M9 3h1" />
  <path d="M9.995 21a2 2 0 0 1-1.727-1" />
```

Reply:
you replaced the wrong icon lol, I'll go in and switch them real quick